### DISCUSSION TITLE
Change Expander Header background without changing the whole style

### DISCUSSION THREAD

**Discussion by steam3d** [microsoft/microsoft-ui-xaml#8872] [Q&A] [answered]
Is there simple way to change header background of expander when hovering mouse on it?

Expander has such StaticResource:
```
 <StaticResource x:Key="ExpanderHeaderBackground" ResourceKey="CardBackgroundFillColorDefaultBrush" />
 ```
 
https://github.com/microsoft/microsoft-ui-xaml/blob/8027ff4af619eb470fab63bf4609c6404ee73e17/dev/Expander/Expander_themeresources.xaml#L14C28-L14C28

I think maybe somehow override it while mouse hovering.

**Comment by castorix** [comment, marked as answer]
A way can be with _PointerEntered_/_PointerExited_
For example, if I change  the background of "ToggleButtonGrid" (got from Visual Tree) to Orange :

![Expander_Header_Color](https://github.com/microsoft/microsoft-ui-xaml/assets/22345506/9cd526d6-33d5-4362-b7f2-78ecb8f7840d)
